addons/test_mail/static/tests/activity_tests.js

```diff
@@ -200,6 +200,47 @@ QUnit.module("test_mail", {}, function () {
         );
     });
 
+    QUnit.test(
+        "activity view: there is no default limit of 80 in the relationalModel",
+        async function (assert) {
+            const mailActivityTypeIds = pyEnv["mail.activity.type"].search([]);
+
+            const recordsToCreate = [];
+            const activityToCreate = [];
+
+            for (let i = 0; i < 81; i++) {
+                activityToCreate.push({
+                    display_name: "An activity " + i,
+                    date_deadline: moment().add(3, "days").format("YYYY-MM-DD"), // now
```

Comment:
Better use `patchDate()`, so the test behaves always the same regardless of when it's run.

Having tests relying on now makes it very prone to fail in the future, e.g. nightly builds or during some periods of the day (Ascension Day for example)